### PR TITLE
Add yaml files

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
This is the first cut at yaml files for gfs RadMon plots.  Included are the time series and summary plots.

Note that I'd like to figure out a way to use yaml anchors and aliases to define satellite, instrument, and valid date/time once in the files.  So far I haven't been able to figure it out, at least within the eva framework.  I'll open an issue to address that.

### CODE REVIEW

File: parm/gfs/monRadSummary.yaml [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+# ---------
+datasets:
+  - name: summary
+    satellite: g16
+    sensor: abi
+    type: MonDataSpace
+    control_file:
+      - ../data/rad_data/time.abi_g16.ctl
+    filenames:
+      - ../data/rad_data/time.abi_g16.2023070218.ieee_d
```
Comment (CoryMartin-NOAA) [inline parm/gfs/monRadSummary.yaml:22]: I'm realizing what our first 'infrastructure' tools will be, to generate this list from a script.

**Review by CoryMartin-NOAA — COMMENTED** [review]
Question for you @EdwardSafford-NOAA , do you prefer to have this committed so that we have it saved and then generalize it, or generalize it now? I'm guessing the former. We should talk soon about the tools the workflow team is developing to fill in dates and variables at runtime with templated YAMLs.

File: parm/gfs/monRadSummary.yaml (hunk 1 above)
Comment (EdwardSafford-NOAA) [reply parm/gfs/monRadSummary.yaml:22]: Yup, that's certainly high on the list.

**Comment by EdwardSafford-NOAA** [comment]
It's probably best to get these under version control and then update them.  I look forward to learning more about the workflow team is doing for yaml completion.  I was trying to figure that out with yaml native solutions, but found they are both quite limited, and don't work within eva.

**Review by CoryMartin-NOAA — APPROVED** [review]
looks good, let's get this saved and then we can work on generalizing/templating. Thanks!

**Pull request merged**